### ISSUE TITLE
Support for multiple assignees

### ISSUE THREAD

**Issue by craig-davis** [ndaversa/hubot-github-bot#7]
This currently appears to use only the first assignee on a PR. Does the current architecture allow this to be extended to:
1. Use the GitHub multiple assignee feature
2. Send a direct message to each person rather than a general announcement

Thoughts?

**Comment by ndaversa** [comment]
@craig-davis You're right. I think the code could be adapted to support multi-assignees (which as you know, is a recent feature Github introduced). I'm happy to help if you are interested in taking a stab at that.

Also happy to accept pull requests for the support.

**Comment by craig-davis** [comment]
@ndaversa Multiple assignee support would be a great addition at work. I'll try to find some time to work on it next week.
